cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -491,7 +491,9 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function (done) {
+        xit('should have appropriate class names on focus/blur', function (done) {
```

Comment:
Should it just be deleted? I investigated several focus tests in this PR, and from what I saw, it is a losing battle to try to unit test focus changes. I didn't see any evidence that it would change in future releases of JQuery (JQuery is deleting their own focus tests). Therefore I'd recommend simply deleting the test.